satellite/analytics/hubspot.go

```diff
@@ -0,0 +1,147 @@
+// Copyright (C) 2021 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package analytics
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/spacemonkeygo/monkit/v3"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+)
+
+var mon = monkit.Package()
+
+type Event struct {
+	Data     map[string]interface{}
+	Endpoint string
+}
+
+type HubspotEvents struct {
+	events        chan []Event
+	apiKey        string
+	satelliteName string
+	log           *zap.Logger
+}
+
+func NewHubSpotEvents(apiKey, satelliteName string, chanSize int) *HubspotEvents {
```

Comment:
I'm starting to think of more config values that might be useful, e.g. we could make the "pe20293085" prefix a config value (would be better in my opinion, because other people may run their own satellites in the future). You don't need to do that here, but one suggestion would be to make a new Config struct in this file
Like 
```
type Config struct {
    APIKey   string `help:"hubspot api key" default:""`
    ChannelSize string `help:"the number of events that can be in the queue before dropping" default:10000`
    EventPrefix string `help:"the prefix for events to send to hubspot" default:"pe20293085"
    ....
}
```
And you can put it in the analytics service config like this
```
type Config struct {
    ...
    Hubspot HubspotConfig
}
```
And add it to the HubspotEvents struct like 
```
type HubspotEvents struct {
    config HubspotConfig
    ....
}
```
Then you can set it like
```
func NewHubSpotEvents(config HubspotConfig ....
    return &HubspotEvents{
        config: config,
        ...
    }
}
```